scripts/qa_crowbarsetup.sh

```diff
@@ -2531,21 +2531,27 @@ function custom_configuration
             fi
             if [[ $want_ldap = 1 ]] ; then
                 local p="proposal_set_value keystone default"
-                if iscloudver 6plus; then
+                if iscloudver 7plus && grep -q domain_specific_config $pfile; then
+                    $p "['attributes']['keystone']['domain_specific_drivers']" true
+                    local l="['attributes']['keystone']['domain_specific_config']['ldap_users']['ldap']"
+                elif iscloudver 6; then
                     $p "['attributes']['keystone']['identity']['driver']" "'hybrid'"
                     $p "['attributes']['keystone']['assignment']['driver']" "'hybrid'"
+                    local l="['attributes']['keystone']['ldap']"
                 else
                     $p "['attributes']['keystone']['identity']['driver']" "'keystone.identity.backends.hybrid.Identity'"
                     $p "['attributes']['keystone']['assignment']['driver']" "'keystone.assignment.backends.hybrid.Assignment'"
+                    local l="['attributes']['keystone']['ldap']"
                 fi
-                local l="['attributes']['keystone']['ldap']"
                 $p "$l['url']" "'ldap://ldap.suse.de'"
+                $p "$l['user']" "'ou=people,dc=suse,dc=de'"
                 $p "$l['suffix']" "'dc=suse,dc=de'"
+                $p "$l['query_scope']" "'sub'"
                 $p "$l['user_tree_dn']" "'ou=accounts,dc=suse,dc=de'"
                 $p "$l['user_objectclass']" "'posixAccount'"
-                $p "$l['user_id_attribute']" "'suseid'"
+                $p "$l['user_id_attribute']" "'uid'"
                 $p "$l['user_name_attribute']" "'uid'"
-                $p "$l['use_tls']" "true"
+                $p "$l['use_tls']" "false"
```

Comment:
Actually, my mistake - use_tls and ldap:// together are compatible and intentional